data/gegno/engines.txt

```diff
@@ -0,0 +1,225 @@
+
+outfit "RG3 Torch Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	cost 128000
+	mass 83
+	thumbnail "outfit/large torch"
+	"engine capacity" -83
+	"outfit space" -83
+	thrust 28.4
+	"thrusting energy" 3.1
+	"thrusting heat" 6.6
+	"flare sprite" "effect/ion torch/large"
+		"frame rate" 2.3
+	"flare sound" "ion torch large"
+	description "RG3 Ion Torch Thrusters are the Gegno's largest thrusters and the oldest still used. Based off of the massive thrusters that powered early Gegno generation ships, these are only found on the largest of Gegno spacecraft, overweighing and overheating anything smaller out of the option."
+
+outfit "RG3 Torch Steering"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	cost 102000
+	mass 64
+	thumbnail "outfit/large torch steer"
+	"engine capacity" -64
+	"outfit space" -64
+	turn 700
+	"turning energy" 1.6
+	"turning heat" 4.2
+	"steering flare sprite" "effect/ion torch/large"
+		"frame rate" 2.3
+	"steering flare sound" "ion torch large"
+	description "As the largest and oldest steering system seen around Gegno space, only a few ships are equipped with RG3 Torch Thrusters, as most ships are nowhere near massive enough to be able to run them."
+
+outfit "SC-1 Plasma Engines"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+		"Scin Architect"
+	cost 128000
+	mass 24
+	thumbnail outfit/tinyscplasma
+	"engine capacity" -21
+	"outfit space" -21
+	thrust 5.1
+	"thrusting energy" 0.54
+	"thrusting heat" 1.2
+	turn 160
+	"turning energy" 0.31
+	"turning heat" 0.86
+	"flare sprite" effect/scplasma/tiny
+		"frame rate" 0.6
+	"flare sound" "scplasma tiny"
+	description "In order to power the small Halite frame, the Gegno Scin developed this smaller engine system. Based on their larger plasma thrusters, these engines are much more efficient than the Vi's larger, more sluggish Ion Engines, which would never be compact or light enough to run on a fighter."
+
+outfit "SC-12 Plasma Thruster"
+	category Engines
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 98000
+	mass 25
+	thumbnail outfit/smallscplasma
+	"engine capacity" -25
+	"outfit space" -25
+	thrust 10.6
+	"thrusting energy" 0.8
+	"thrusting heat" 2.7
+	"flare sprite" effect/scplasma/small
+		"frame rate" 0.5
+	"flare sound" "scplasma small"
+	description "Rather than further developing on the Ion Torch design of the Gegno Vi, the Gegno Scin began work on a plasma-based thruster concept. They are more compact, weigh less, and produce significantly more thrust. However, since Ion Torches expel a percentage of heat in their thrust, the more accurate and powerful SC thrusters tend to run notably hotter."
```

Comment:
Accurate? Not sure that the meaning of that is entirely clear in relation to a thruster.
Is this in relation to the ion engines having "chaotic" output?
(From: `	description "Ion Torch Thrusters are the Gegno's equivalent of humanity's ion thrusters, but are still earlier in design and lacking in most departments when compared. They have heavier frames and their thrust output is a lot more chaotic and explosive, causing them to run notably hotter but require slightly less energy. The RG18 is the most common-sized thruster amongst Gegno Vi vessels, and a notable size at that."`, the RG18 thruster description.)
Perhaps 'consistent' or 'stable' instead of 'accurate' if that's the case?
Also, this engine offers a thrust/heat of 3.93, actually marginally higher than the 3.84 of the (slightly larger) RG15 thruster.

Reply:
Applying stable in place of accurate. Yes, it was in relation to the torches having "chaotic" output. Will look into raising the heat as well after looking at other comments in data you've made, or changing the description to just say "tend to run hotter".